playbook/appherd/roles/create_env_settings/templates/aws-env.py.j2

```diff
@@ -98,42 +98,8 @@ SEND_SMS = True
 #         return HttpResponseRedirect(reverse('login'))
 MFA = True
 
-# Add in apps.accounts backends for DEV environment
-#AUTHENTICATION_BACKENDS = (
-#    'django.contrib.auth.backends.ModelBackend',
-#    'apps.accounts.auth.SettingsBackend',
-#    # 'apps.accounts.mymedicare_auth.MyMedicareBackend',
-#)
-AUTHENTICATION_BACKENDS = {{ django_authentication_backends }}
```

Comment:
It looks like this was set to both the `ModelBackend` and `SettingsBackend` but base.py only includes `ModelBackend` is that a problem?

Reply:
So, the real reason for this change is that I suspect SettingsBackend - which hasn't been used, updated, or tested in about a year, might be causing issues. I'm purposefully removing it because it's untested code that does nothing and might cause problems.